doc/classes/DisplayServer.xml

```diff
@@ -776,14 +776,14 @@
 			<return type="bool" />
 			<description>
 				Returns [code]true[/code] if OS is using dark mode.
-				[b]Note:[/b] This method is implemented on macOS, Windows and Linux (X11).
+				[b]Note:[/b] This method is implemented on Android, iOS, macOS, Windows and Linux (X11).
```

Comment:
```suggestion
				[b]Note:[/b] This method is implemented on Android, iOS, macOS, Windows, and Linux (X11).
```
Oxford comma

Reply:
I'm not a native English speaker, but should I also use a comma before `and`? See other documents.

Reply:
See [here](https://docs.godotengine.org/en/stable/contributing/documentation/docs_writing_guidelines.html#use-the-oxford-comma-to-enumerate-anything)

Reply:
But don't change any of the other documentation here